pkg/gateway/model_build_targets.go

```diff
@@ -119,15 +138,19 @@ func (t *latticeTargetsModelBuildTask) buildLatticeTargets(ctx context.Context)
 		glog.V(6).Infof("Build Targets:  endPoints %v \n", endPoints)
 
 		for _, endPoint := range endPoints.Subsets {
-
 			for _, address := range endPoint.Addresses {
 				for _, port := range endPoint.Ports {
 					glog.V(6).Infof("serviceReconcile-endpoints: address %v, port %v\n", address, port)
 					target := latticemodel.Target{
 						TargetIP: address.IP,
 						Port:     int64(port.Port),
 					}
-					targetList = append(targetList, target)
+					if portAnnotations == 0 || int64(target.Port) == portAnnotations {
+						targetList = append(targetList, target)
+						fmt.Printf("portAnnotations:%v, target.Port:%v\n", portAnnotations, target.Port)
```

Comment:
can u use glog.V(6).Infof here instead of fmt.Printf()

Reply:
Done